### ISSUE TITLE
Missing data when using query_generation() and query_load()

### ISSUE THREAD

**Issue by mechen-gess** [EnergieID/entsoe-py#158]
Dear Developers,
 
Thank you very much for providing the possibility to extract ENTSO-E data conveniently. Lately I have encountered an issue: when I was trying to use query_generation() or query_load() to get the data of a certain year, for example 2015, the outcome is not complete, as the file attached. It is supposed to be half-hourly resolution, but some half-hours are missing, as you can already see in the first 3 hours. When one goes to the API of ENTSO-E Transparency Platform, however, one sees the complete data. A colleague of mine used the version 0.3.2, while I use the latest 0.4.3, and there is higher chance that he extracts the complete data. I was wondering where the problem could lie, and if you have encountered similar issue and aware of the solution. Any help would be appreciated. Thank you very much in advance!

[GB_Generation_byType_2015v1.csv](https://github.com/EnergieID/entsoe-py/files/8022724/GB_Generation_byType_2015v1.csv)

Best regards,
M.C.

**Comment by MattEwen** [comment]
I'm not certain but I suspect the issue is here - in changes to `decorators.py` in PR #148. It looks like this changes the drop duplicates behaviour to drop duplicate values rather than indices. It would make sense that this shows up obviously in series, but rarely in dfs - as in dfs there will be enough columns that complete duplicate rows would be rare

<img width="2218" alt="Screenshot 2022-02-17 at 07 22 37" src="https://user-images.githubusercontent.com/71700970/154481008-2be3807d-eb1a-488d-ad49-739ae813201f.png">

**Comment by fboerman** [comment]
hi @MattEwen  thank you for reporting this. this is indeed a problem with that decorator which was changed in a merged PR which I slipped past my checking.

there should be a duplicated check there but it should only look at the index, that is how it always has been. but the PR changed it to a duplicated call on the dataframe or series itself which means it deletes duplicated prices instead of indexes. a subtle but important difference. I will revert that line and push a new version!
